### ISSUE TITLE
Why are there two annotation objects afterwards?

### ISSUE THREAD

**Issue by JohnPower2k** [Phlya/adjustText#86]
I would like to be able to move the texts around manually after adjustText did its job. Also, I have defined arrows to connect the texts to points. 
So what I tried was to iterate over all the annotations and calling .draggable(True). 
Unfortunately, I've discovered that for every Text Object, two Annotation Objects are created by AdjustText... one for the actual text and one for the arrow.
This is problematic because when moving the Annotations around, the arrow stays in place. Is there a special reason for creating two annotation objects? Couldn't you just add the arrow to the annotation object that holds the actual text?

**Comment by Phlya** [comment]
Good question! Originally it was implemented as you are suggesting, with a single new Annotation object. However, seems like there is a subtle/difficult to reproduce bug in matplotlib that sometimes this sort of thing happened: https://github.com/Phlya/adjustText/issues/13 And it seems, that after I switched to the current implementation, this problem disappeared - however without knowing the exact cause of that problem I am not sure exactly what happened. I'm happy to discuss returning the old implementation, if this doesn't cause that sort of issues.

By the way, there even used to be a draggable keyword in adjust_text to make all annotations draggable, but the dragging was really buggy and I removed it - has it been fixed in mpl, does it work well for you?
